samples/Config.CustomConfigurationProviders.Web/Program.cs

```diff
@@ -0,0 +1,44 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.ConfigFile;
+using System;
+
+namespace Config.CustomConfigurationProviders.Web
```

Comment:
This project isn't actually a web site, so it probably shouldn't be named `*.Web`.

Perhaps the config provider project should be `Config.ConfigFileProvider`, and this sample app should be `Config.ConfigFileProvider.Sample`?